stubs/docutils/docutils/nodes.pyi

```diff
@@ -10,8 +11,27 @@ class Node:
     def __getattr__(self, __name: str) -> Any: ...  # incomplete
 
 class Element(Node):
+    children: list[Node]
     def __init__(self, rawsource: str = ..., *children: Node, **attributes): ...
     def __getattr__(self, __name: str) -> Any: ...  # incomplete
```

Comment:
Nit: I think it would be clearer to move this to the bottom, as its basically an "... etc.".